src/reducers/index.ts

```diff
@@ -109,9 +132,23 @@ const draftReducer = (draft: Post | null = null, action: Actions): Post | null =
     return draft;
 };
 
+const metadataReducer = (metadata: Metadata = { highestSeenId: 0 }, action: Actions): Metadata => {
+    switch (action.type) {
+        case 'UPDATE-HIGHEST-SEEN-ID': {
+            return {
+                ...metadata,
+                highestSeenId: metadata.highestSeenId++,
```

Comment:
`metadata.highestSeenId + 1` would be nicer here, because the `++` has also an unintended side effect.

Reply:
👍